repairnator/src/main/java/fr/inria/spirals/repairnator/process/step/BuildShouldFail.java

```diff
@@ -20,11 +21,17 @@ public boolean shouldBeStopped(GatherTestInformation gatherTestInformation) {
                     // its previous build is a failing build with failing tests
                     // and it can also be reproduced
                     gatherTestInformation.setState(ProjectState.FIXERBUILD_CASE1);
+                    if (inspector instanceof ProjectInspector4Bears) {
```

Comment:
I really don't like that, but I don't have an easy solution yet. So if you have ideas to avoid this cast...